### PR TITLE
mine_and_craft sript updated. Again

### PR DESCRIPTION
**Pull request by DaddyWesker**
So, i've splitted mine_and_craft.py to vereya and old version as Oleg asked.

### CODE REVIEW

File: examples/4_mine_and_craft.py [hunk 1]
```diff
@@ -47,26 +48,28 @@ def runStraight(rob, dist, keepHeight=False):
             bJump = False
         los = rob.getCachedObserve('getLineOfSights')
         if los is not None and los['distance'] < 0.5 and \
-           not los['distance'] in RobustObserver.passableBlocks and \
-           not los['type'] in RobustObserver.passableBlocks and\
-           not bJump:
+                not los['distance'] in RobustObserver.passableBlocks and \
```
Comment (noskill) [inline examples/4_mine_and_craft.py:51]: what does it check `los['distance'] in RobustObserver.passableBlocks` ???

File: examples/Vereya/4_mine_and_craft.py [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+            rob.sendCommand("jump 1")
+            rob.sendCommand("attack 1")
+            bJump = True
+        elif bJump:
+            rob.sendCommand("jump 0")
+            rob.sendCommand("attack 0")
+            bJump = False
+        los = rob.getCachedObserve('getLineOfSights')
+        if los is not None and los['distance'] < 0.5 and \
+               not los['distance'] in RobustObserver.passableBlocks and \
```
Comment (noskill) [inline examples/Vereya/4_mine_and_craft.py:64]: `not los['distance'] in RobustObserver.passableBlocks` what is checked here?

**Comment by DaddyWesker** [comment]
@noskill 
Anatoly, function runStraight haven't been changed by me at all (except some tabs and spaces). So you should ask about this someone else who wrote it in the first place.

**Review by CICS-Oleg — APPROVED** [review]

File: examples/Vereya/4_mine_and_craft.py (hunk 2 above)
Comment (noskill) [reply examples/Vereya/4_mine_and_craft.py:64]: los['distance'] is a floating point number, no?

Comment (CICS-Oleg) [reply examples/Vereya/4_mine_and_craft.py:64]: Ah, I see your point. It's very strange. It seems like this bug lasts even from the first Alexey's commits... Ironically it hasn't affected performance of the script yet. @DaddyWesker We can discuss how to fix this.

Comment (noskill) [reply examples/Vereya/4_mine_and_craft.py:64]: `not los['distance'] in RobustObserver.passableBlocks` is always True

Comment (DaddyWesker) [reply examples/Vereya/4_mine_and_craft.py]: @noskill  we've decided to remove this line. I've commited new changes.

**Review by noskill — APPROVED** [review]

**Pull request merged**